tasks/workers/etl/sandbox/etl.js

```diff
@@ -64,14 +64,54 @@ function getUrlFromNodeId(node){
   });
 }
 
+function getDateId(date,interval){
+  return interval == "day" ? date.toLocaleDateString():date.toLocaleDateString()+ ":" + date.getHours();
+}
+
+function buildTimeseries(nodes){
+  var dateMap = {};
+  var firstDate;
+  for(var i =0; i< nodes.length; i++){
+    var node = nodes[i];
+    var date = new Date(node.time * 1000);
+    if(!firstDate || node.time < firstDate){
+      firstDate = node.time;
+    }
+    var dateToHour =  new Date(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours());
```

Comment:
@justinlueders moment.js (which we have in the project) helps a lot with date-time conversion tasks like this. fyi... less hackery with native js Date methods. :clock1: